gnat2goto/driver/tree_walk.ads

```diff
@@ -76,4 +76,5 @@ package Tree_Walk is
      return Symbol
    with Pre => Nkind (N) = N_Compilation_Unit;
 
+   type Irep_Array is array (Positive range <>) of Irep;
```

Comment:
This is an example of a scenario where a commit comment as to why this type was introduced would have been really helpful. 
A reviewer must now look at the locations where this is used and reverse engineer why it is needed / intent.

Reply:
Added.